saleor/graphql/page/tests/test_page.py

```diff
@@ -956,25 +997,27 @@ def test_create_page_with_product_reference_attribute_required_no_references_giv
     assert errors[0]["attributes"] == [file_attribute_id]
 
 
-def test_page_delete_mutation(staff_api_client, page, permission_manage_pages):
-    query = """
-        mutation DeletePage($id: ID!) {
-            pageDelete(id: $id) {
-                page {
-                    title
-                    id
-                }
-                pageErrors {
-                    field
-                    code
-                    message
-                }
-              }
+PAGE_DELETE_MUTATION = """
```

Comment:
Could you reformat this mutation?
It's easier to read when mutations and queries have consistent formatting.